deploy/crds/kustomize/patches.json

```diff
@@ -0,0 +1,23 @@
+[
+    {
+        "op": "add",
+        "path": "/spec/versions/0/schema/openAPIV3Schema/properties/spec/properties/maximumCADuration/pattern",
+        "value": "^[0-9][0-9hmnsu.]+"
```

Comment:
Technically, some other characters are allowed (micro and mu symbols for microseconds). But honestly it might be more confusing to include those here. And they aren't included in the comment in the go std library code anyway: https://cs.opensource.google/go/go/+/refs/tags/go1.17.3:src/time/format.go;l=1510

Maybe adjust the regex so it has to end with one of the letters? The current regex allows just a number like `12`.

Reply:
Actually, not just any letter - specifically `[hms]` I think

Reply:
Since the original PR's intent was to add back in what was already there, I added the update to the regex in a separate commit.